Runtime/Scripts/ExampleRest.cs

```diff
@@ -22,6 +23,11 @@ IEnumerator Start()
         async void GetEdgeConnection()
         {
             mxi = new MobiledgeXIntegration();
+            var deviceInfo = await mxi.GetDeviceInfo();
+            foreach (KeyValuePair<string, string> pair in deviceInfo)
+            {
+                Debug.Log("DeviceInfo: Key is " + pair.Key + ", and Value is " + pair.Value);
+            }
```

Comment:
I think you might forgot this here when you were debugging, but I don't think this should be here ?

Reply:
ya i can remove